applications/Chat/tests/test_models.py

```diff
@@ -129,12 +131,12 @@ def test_lora(lora_rank: int,
     # HACK: skip llama due to long execution time
     # lambda: (LlamaActor(), LlamaCritic(), LlamaRM()),
     lambda: (OPTActor(), OPTCritic(), OPTRM()),
+    lambda: (ChatGLMActor(), None, None),
 ])
 @torch.no_grad()
-def test_models(models_maker: Callable[[], Tuple[Actor, Critic, RewardModel]],
+def test_models(models_maker: Callable[[], Tuple[Actor, Critic, RewardModel, Actor]],
```

Comment:
Why do we need to add one more Actor here

Reply:
fixed